AS5045.cpp

```diff
@@ -184,22 +185,31 @@ boolean AS5045::init ()
   
 }
 
+void AS5045::clock_cycle ()
+{
+  if (_clockDelay == 0)
+    return;
```

Comment:
there should be a pinCLK LOW -> HIGH before return to init reading for `clockDelay == 0`
```suggestion
  if (_clockDelay == 0)
  	digitalWrite (_pinCLK, LOW);
  	digitalWrite (_pinCLK, HIGH);
    return;
```

Reply:
Yikes, I had originally put the delay in its own function, then refactored.  I'll submit an updated PR.